### DISCUSSION TITLE
Code review IM-11/13

### DISCUSSION THREAD

**Discussion by AidXylelele** [HowProgrammingWorks/LiveQA#58] [Code review]
## Курсовая работа!

- #### 1 курс 2 семестр;
### Участники: 

1. [Martynenko Oleh IM-11](https://github.com/olegoofy)
2. [Chyrkov Maksym IM-11](https://github.com/chyrkovmaksym)
3. [Shevyrov Vladislav IM-11](https://github.com/AidXylelele)
4. [Anenko Ivan IM-13](https://github.com/LikerFeed)
5. [Andrukhiv Vita IM-13](https://github.com/vita133)

Мы занимаемся разработкой web-сервиса для быстрого создания блок-схем кода, написанного на языке C.
Основа проекта - парсер кода С.
Визуализация - HTML Canvas(в разработке).
В проекте не используются сторонние библиотеки и фреймворки.
Используется метод Fetch для реализации отображения аватарки пользователя и ссылки на его Github аккаунт, совершившего последний commit в наш репозиторий!

## [Ссылка на репозиторий нашего проекта.](https://github.com/chyrkovmaksym/course_project.git)
@tshemsedinov

**Comment by tshemsedinov** [comment]
https://github.com/chyrkovmaksym/course_project/issues/1
